src/site-layout/SiteLayout.js

```diff
@@ -1,16 +1,33 @@
 import useBreakpoint, { mediaQuery } from "@w11r/use-breakpoint";
 import PropTypes from "prop-types";
 import React from "react";
-import styled from "styled-components";
+import styled, { css } from "styled-components";
+import { useLocation } from "@reach/router";
 import BrandingBar from "../branding-bar";
-import { CONTAINER_WIDTH, FIXED_HEADER_HEIGHT, X_PADDING } from "../constants";
+import {
+  CONTAINER_WIDTH,
+  FIXED_HEADER_HEIGHT,
+  PATHS,
+  X_PADDING,
+} from "../constants";
 import Footer from "../footer";
 import NavBar from "../nav-bar";
 import PageRoutes from "../page-routes";
+import BackgroundImageSrc from "../assets/images/background.png";
```

Comment:
I'm not sure if this background image is meant to be global or branded per tenant but I punted on that concern for the time being.